electron/main.ts

```diff
@@ -433,7 +434,7 @@ ipcMain.on('Notify', (event, args) => {
 })
 
 ipcMain.on('frontendReady', () => {
-  handleProtocol(mainWindow, process.argv)
+  handleProtocol(mainWindow, [openUrlArgument, ...process.argv])
```

Comment:
because of how handleProtocol works, this is fine, since it will go over all the elements of this array until if finds a heroic procotol, so if this is an empty string it's just skipped